mmcv/image/io.py

```diff
@@ -239,20 +238,46 @@ def imfrombytes(content, flag='color', channel_order='bgr', backend=None):
         return img
 
 
-def imwrite(img, file_path, params=None, auto_mkdir=True):
+def imwrite(img,
+            file_path,
+            params=None,
+            auto_mkdir=True,
+            file_client_args=None):
     """Write image to file.
 
+    Note:
+        In v1.3.17 and later, add `file_client_args` parameters.
```

Comment:
Maybe v1.3.17 --> v1.3.19

Reply:
Yes, I will modify this problem in the next commit.